v63002/outref/gtm8711.txt

```diff
@@ -1,21 +1,12 @@
-spawn /usr/local/bin/tcsh -f
-> # Expect the shell prompt
-stty cols 132
-# Invoking GDE Directly From Shell
-$ydb_dist/mumps -run ^GDE
-> $ydb_dist/mumps -run ^GDE
-%GDE-I-GDUSEDEFS, Using defaults for Global Directory 
-	##TEST_PATH##/mumps.gld
-
-GDE> # Sending Invalid Command, Expect The Shell To Maintain The Error Status
-asdfasdfasdf
-%GDE-E-KEYWRDBAD, ASDFASDFASDF is not a valid verb in this context
+# Running expect script in which we enter GDE and run an invalid command
+error status=114
```

Comment:
I would also like to see the KEYWRDBAD error in this case in the reference file.